### DISCUSSION TITLE
Előadáson elhangzott

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#20] [Kérdések és válaszok]
A 2. előadás 17. dián  (REST operáció) mit jelent az "adózói adat lekérdezés"?

**Comment by vectorkovacspeter** [comment]
@MicraMetripond az előadás anyag, amire hivatkozol hol érhető el?

**Reply by MicraMetripond** [reply]
https://nav.gov.hu/kereso?query=e-nyugta

**Reply by vectorkovacspeter** [reply]
köszi, ez megvan nekem is, csak azt hittem van egy másik is, mert ebben sem a 17 dia, sem másik nem tartalmaz ilyet a 03.07 -én publikált verziójában.
